com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -320,8 +318,6 @@ internal DebugDisplaySettings()
                 s_RenderingFullScreenDebugValues = s_RenderingFullScreenDebugValues.Where((val, idx) => (idx + FullScreenDebugMode.MinRenderingFullScreenDebug) != FullScreenDebugMode.QuadOverdraw).ToArray();
             }
 
-            FillTileClusterDebugEnum();
```

Comment:
Why remove this? As I see we were replacing part of the strings compared to original enum names so we are losing that here.

Reply:
Did not see the InspectorName attribute. Disregard this comment :)